src/math/bessel.hh

```diff
@@ -200,7 +200,9 @@ namespace rascal {
           this->upward_recursion(distance, fac_a, n_up);
         }
 
-        // set small values to 0.
+        // set small values to 0. On the python side it avoids some unexpected
+        // interpretation of values that are strictly speaking outside of the
+        // range of double precision
```

Comment:
I don't understand this comment. 1e-100 is still withing the range of double precision, and pretty far from the limit (1e-308). What do you mean by `unexpected interpretation`?

Reply:
1e-100 is because we anyway don't care for smaller stuff and you can get values ~1e-312 which on the python side are not well handled by json serialization for instance.